esp/services/esdl_svc_engine/esdl_binding.hpp

```diff
@@ -63,7 +64,11 @@ static const char* ESDL_METHOD_HELP="help";
 #define REQ_REF_NUM_NAME    "_req_ref_num"
 #define MCACHE_OBJECT_KEY   "_mcache_object_key_"
 
+typedef IEmbedContext * (* GetEmbedContextFunction)();
```

Comment:
Possibly move to hqlplugins.hpp, so that the same mechanism can be used for other language plugins.